handbook/engineering/onboarding.md

```diff
@@ -5,7 +5,8 @@ Welcome to Sourcegraph! This document will guide you through engineering specifi
 ## Manager checklist
 
 - Grant access to necessary services.
-  - [Sourcegraph organization on GitHub](https://github.com/sourcegraph/people)
+  - [Sourcegraph organization on GitHub](https://github.com/orgs/sourcegraph/people)
+  - [LSIF organization on GitHub](https://github.com/orgs/lsif/people) (optional; recommended for Code Intelligence team members)
```

Comment:
Who is on that team? Can you invite me?

Reply:
I don't know. I wrote the PR because I'm not one of them. 😀